_posts/elements/2015-07-01-grid.md

```diff
@@ -0,0 +1,878 @@
+---
+layout:       element
+title:        Grid System
+date:         2015-07-01 00:00:00
+
+categories:   elements
+tags:
+- atomic
+- pattern
+- grid
+
+slug:         grid
+scssurl:      "https://github.com/edx/ux-pattern-library/blob/master/_src/pattern-library/sass/components/_grid.scss"
+
+description:  Grid system and settings that can be used throughout layouts.
+
+info: A responsive, mobile first fluid grid system, based on Susy and Breakpoint, that appropriately scales up to 12 columns as the device or viewport size increases. It includes some predefined classes for easy layout options, while providing Sass utilities for generating more semantic layouts.
+
+example: grid.html
+---
+
+<h3 class="hd-6 example-set-hd">Basic Grid</h3>
+<div class="example-set">
+    <div class="example-grid example-grid-basic">
+        <div class="grid-container">
+            <div class="col first">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+               <span class="example-label">Col</span>
+            </div>
+
+            <div class="col">
+                <span class="example-label">Col</span>
+            </div>
+
+            <div class="col last">
+                <span class="example-label">Col</span>
+            </div>
+        </div>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Basic Grid (with column shifts)</h3>
+<div class="example-set">
+    <div class="example-grid example-grid-shifts">
+        <div class="grid-container">
+            <section class="shifts-pre">
+                <div class="col col-A">
```

Comment:
Oh, no. This is the shift example that uses Sass to do the shifting (vs. classes in the DOM).

You can see the matching demo Sass I've used here - https://github.com/edx/ux-pattern-library/blob/talbs/elements-grid/_src/pldoc/static/sass/_components.scss#L595.

The DOM-based way of shifting is included in the one class-based example I've added - https://github.com/edx/ux-pattern-library/pull/105/files#diff-6637bee6def53234581792fe31fa4301R176.

Adding descriptions to each example will help this type of confusion, I hope.